cmd/start.go

```diff
@@ -35,30 +49,47 @@ func startNetwork(cmd *cobra.Command, args []string) error {
 		snapshotName = args[0]
 		startMsg = fmt.Sprintf("Starting previously deployed and stopped snapshot %s...", snapshotName)
 	} else {
-		snapshotName = defaultSnapshotName
+		snapshotName = constants.DefaultSnapshotName
 		startMsg = "Starting previously deployed and stopped snapshot"
 	}
 
 	ctx := binutils.GetAsyncContext()
 
 	ux.Logger.PrintToUser(startMsg)
-	_, err = cli.LoadSnapshot(ctx, snapshotName)
+	loadSnapshotOpts := []client.OpOption{
+		client.WithPluginDir(pluginDir),
+		client.WithExecPath(avalancheGoBinPath),
+	}
+	_, err = cli.LoadSnapshot(
+		ctx,
+		snapshotName,
+		loadSnapshotOpts...,
+	)
 	if err != nil {
-		return fmt.Errorf("failed to start network with the persisted snapshot: %s", err)
+		if !strings.Contains(err.Error(), "already bootstrapped") {
```

Comment:
We are "institutionalizing" this pattern which I don't like much. We should not check for error texts to identify error conditions.

If you don't want to change this in this PR, we should have a TODO and ticket and do it later (soon).

Reply:
Done both. https://ava-labs.atlassian.net/browse/AV-1749